install.sh

```diff
@@ -1,24 +1,38 @@
 #!/bin/bash
 
 # un-comment to install anaconda
-install_dir=$HOME/anaconda3
-wget https://repo.anaconda.com/archive/Anaconda3-2020.11-Linux-x86_64.sh
-bash Anaconda3-2020.11-Linux-x86_64.sh -b -p  $install_dir
-export PATH=$install_dir/bin:$PATH
+
+isPackageNotInstalled() {
+    $1 --version &> /dev/null
+    if [ $? -eq 0 ]; then
+    echo "$1: Already installed"
+    else 
+    install_dir=$HOME/anaconda3
+    wget https://repo.anaconda.com/archive/Anaconda3-2020.11-Linux-x86_64.sh
+    bash Anaconda3-2020.11-Linux-x86_64.sh -b -p  $install_dir
+    export PATH=$install_dir/bin:$PATH    
+    fi
+}
+
+isPackageNotInstalled conda
 
 
 # create conda env
+export FEDSCALE_HOME=$PWD
```

Comment:
Move this upward in a section on its own as it's not related to conda and out of place here.